e2e/libs/node/node.py

```
@@ -9,15 +10,27 @@
     def __init__(self):
         self.retry_count, self.retry_interval = get_retry_count_and_interval()
 
+    def mount_disk(self, disk_name, node_name):
+        mount_path = os.path.join(self.DEFAULT_DISK_PATH, disk_name)
+        device_path = os.path.join(self.DEFAULT_VOLUME_PATH, disk_name)
+        cmd = f"mkdir -p {mount_path}"
+        res = NodeExec(node_name).issue_cmd(cmd)
+        cmd = f"mkfs.ext4 {device_path}"
+        res = NodeExec(node_name).issue_cmd(cmd)
+        cmd = f"mount {device_path} {mount_path}"
+        res = NodeExec(node_name).issue_cmd(cmd)
```

Ruff

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)